### PR TITLE
Fixes #118 : Facebook account addition UI fix

### LINKED ISSUE #118: Facebook account addition UI fix
**Actual Behaviour**

An additional activity with "continue with facebook" button is opening which is out of given workflow.

**Expected Behaviour**

Facebook should be opened directly on clicking in the dialog box.

**Screenshots of the issue**

screenshot_2017-03-12-01-59-43

**Would you like to work on the issue?**

Yes I would like to work on it.

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #118 

Change - Direct continuing with facebook instead of an additional activity
Screenshots for the change: 
screenshot_2017-03-16-21-17-16

### CODE REVIEW

**Comment by Abhi2424shek** [comment]
Resolve the conflicts pls.

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 1]
```diff
@@ -329,8 +340,85 @@ public void onClick(View v) {
         btnAdd.setOnClickListener(new OnClickListener() {
             @Override
             public void onClick(View v) {
-                DialogFragment newFragment = new AddAccountDialogFragment();
-                newFragment.show(getFragmentManager(), "dialog");
+                AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(ctx);
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: Use `this` instead of ctx.

File: app/src/main/res/layout/dialog_add_account.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        android:paddingRight="30dp"
+        android:paddingLeft="30dp"
+        android:paddingTop="10dp"
+        android:paddingBottom="10dp"
+        android:layout_marginBottom="30dp"
+        android:background="@color/wordpress_add_account"
+        android:text="@string/wordpress"
+        android:textColor="@color/cardview_light_background"
+        android:layout_gravity="center_horizontal" />
+</LinearLayout>
```
Comment (Abhi2424shek) [inline app/src/main/res/layout/dialog_add_account.xml]: Add a new line at EOF

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -24,5 +24,6 @@
     <color name="custom_progress_bg_color">#EEEEEE</color>
     <color name="custom_progress_message_color">#6D6D6D</color>
     <color name="image_selected">#96dde238</color>
+    <color name="wordpress_add_account">#21759b</color>
     <color name="md_dark_primary_text">#FFFFFFFF</color>
 </resources>
```
Comment (Abhi2424shek) [inline app/src/main/res/values/colors.xml]: If possible add a new line here also.

**Comment by rohanagarwal94** [comment]
@Abhi2424shek made the necessary changes. Please review.

File: app/src/main/java/vn/mbm/phimp/me/Upload.java (hunk 1 above)
Comment (rohanagarwal94) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: @Abhi2424shek its a fragment, hence context is required.

File: app/src/main/res/layout/dialog_add_account.xml (hunk 2 above)
Comment (rohanagarwal94) [inline app/src/main/res/layout/dialog_add_account.xml]: Added it.

**Pull request merged**